terraform/modules/gro_ingestion_service/lambdas/src/functions/enrichEvent.function.ts

```diff
@@ -0,0 +1,34 @@
+import {Handler} from "aws-lambda"
+import {DocumentClient} from "aws-sdk/clients/dynamodb"
+import {mapToEventRecord} from "../models/EventRecord";
+import {EnrichEventRequest} from "../models/EnrichEventRequest";
+import {EnrichEventResponse} from "../models/EnrichEventResponse";
+
+const tableName = process.env.TABLE_NAME ?? ""
+
+const dynamo = new DocumentClient({apiVersion: '2012-08-10'})
+
+export const handler: Handler = async (event: EnrichEventRequest, context, callback): Promise<EnrichEventResponse> => {
+
+    const params: DocumentClient.GetItemInput = {
+        Key: {
+            hash: event.id
+        },
+        TableName: tableName
+    };
+
+    const result = await dynamo.get(params).promise()
+
+    if (!result.Item) {
+        console.log(`Record with hash ${event.id} not found`)
+        return {
+            statusCode: 404
+        }
+    }
+
```

Comment:
could we add a log that we have successfully enriched this event

also could we add logParams like we have in https://github.com/alphagov/gdx-data-share-poc/pull/1065/files#diff-2ba6df16e5593d034873a60f8eceb033076b2b9bdaad8af5753ed63c0e5a0cc1 which have the EventTime, RegistrationId, and hash